### DISCUSSION TITLE
svgViewBoxSize missing in v5.0

### DISCUSSION THREAD

**Discussion by manuelKruesi** [chillerlan/php-qrcode#230] [Q&A] [answered]
Hello there! First of all, thanks for this amazign library and the constant support!
I am currently struggling with my generated svg only being 45x45, which is way too small. In the past, this was changed with the svgViewoxSize Option, but this seems missing since v5.0. Is there now another intended way? 
Thanks for the support!

**Comment by codemasher** [comment, marked as answer]
Hi, the `viewboxSize` has been removed because of the common misconception that it would change the size of the QR Image - the viewbox size changes only the canvas size of the SVG, not the size of the contents, see #222. 
If you want to change the output size of the SVG (i.e. when CSS is not available), you need to set the `width` and `height` attributes on the `<svg>` element:
https://github.com/chillerlan/php-qrcode/blob/57448af264a4bc0aca85a9ce61769aa6240b9481/examples/imagickConvertSVGtoPNG.php#L33-L42

**Reply by codemasher** [reply]
(If you still need to change the `viewboxSize`, you just need to override the `getViewBox()` method.)

**Reply by manuelKruesi** [reply]
I see, thanks a lot for the clarification. The Example provided stems from the svg to png conversion, is there a way to directly apply these attributes (width and height) in the MARKUP_SVG, because the end product should still be an svg, not a png! Thanks for your quick help!

**Reply by codemasher** [reply]
You would just need to create your own custom output class and just override the desired method, just like it's shown in the several SVG examples. In fact, you could just use the above example as a basis and remove the `dump()` method, because that's where the conversion magic happens.

**Reply by manuelKruesi** [reply]
thanks a lot and have a nice weekend!

**Reply by codemasher** [reply]
Same to you!
